LibAddonMenu-2.0/LibAddonMenu-2.0/controls/editbox.lua

```diff
@@ -15,12 +17,28 @@
 } ]]
 
 
-local widgetVersion = 14
+local widgetVersion = 15
 local LAM = LibStub("LibAddonMenu-2.0")
 if not LAM:RegisterWidget("editbox", widgetVersion) then return end
 
 local wm = WINDOW_MANAGER
 
+local function GetValidTextType(textType)
+    textType = LAM.util.GetDefaultValue(textType)
+    if textType and (textType >= TEXT_TYPE_ITERATION_BEGIN and textType <= TEXT_TYPE_ITERATION_END) then
+        return textType
+    end
+    return TEXT_TYPE_ALL
+end
+
+local function GetValidMaxChars(number)
+    number = LAM.util.GetDefaultValue(number)
+    if number and type(number) == "number" then

```

Comment:
the type check already eliminates the possibility for a value that evaluates to false, so the first condition can safely be removed.